rust/core/src/driver_manager.rs

```diff
@@ -276,6 +351,23 @@ impl ManagedDriver {
 
         Ok(database)
     }
+
+    fn load_driver_from_manifest(
+        driver_path: &Path,
+        entrypoint: Option<&[u8]>,
+        version: AdbcVersion,
+    ) -> Result<Self> {
+        let mut info = DriverInfo {
+            manifest_file: driver_path.to_path_buf(),
+            driver_name: String::new(),
+            lib_path: PathBuf::new(),
+            entrypoint: entrypoint.map(|e| e.to_vec()),
+            version: String::new(),
+            source: String::new(),
```

Comment:
I think it works to just write `Default::default()` instead of explicitly calling `new`

Reply:
You can also write the type name, i.e. `DriverInfo::default()` - this is also a pedantic clippy lint: https://rust-lang.github.io/rust-clippy/master/index.html#default_trait_access